vulnerabilities/scraper/npm.py

```diff
@@ -0,0 +1,105 @@
+# Author: Navonil Das (@NavonilDas)
+import semantic_version
+import re
+import json
+from urllib.request import urlopen
+
+NPM_URL = 'https://registry.npmjs.org{}'
+PAGE = '/-/npm/v1/security/advisories?page=1'
+    
+# Removes spaces and v Charecter in front of version 
```

Comment:
Could you make this a docstring rather than a comment? Also plase make sure you use pep8-like formatting (e.g. 2 empty lines between top level members)

Reply:
ok, I will change the requested things.